routes/web.php

```diff
@@ -16,3 +17,10 @@
 Route::get('/', function () {
     return view('welcome');
 });
+Route::get('/login',[CustomAuthController::class,'login'])->middleware('alreadyLoggedIn');
```

Comment:
Handig om al je routes een naam te geven, om te gebruiken als named routes.
Zoals al we het geval bij register-user en login-user.
->name('naam van route')